### PR TITLE
use internal client

### PR DESCRIPTION
**Pull request by kmrmt**
### Description:

fix load test using internal client and add option about resolve DNS 

### Related Issue:

#557 

### How Has This Been Tested?:

Nothing.

### Environment:

- Go Version: 1.14.4
- Docker Version: 19.03.8
- Kubernetes Version: 1.18.2
- NGT Version: 1.12.0

### Types of changes:

- [ ] Bug fix [type/bug]
- [X] New feature [type/feature]
- [ ] Add tests [type/test]
- [ ] Security related changes [type/security]
- [ ] Add documents [type/documentation]
- [X] Refactoring [type/refactoring]
- [ ] Update dependencies [type/dependency]
- [X] Update benchmarks and performances [type/bench]
- [ ] Update CI [type/ci]

### Changes to Core Features:

- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully ran tests with your changes locally?

### Checklist:

- [X] I have read the [CONTRIBUTING](https://github.com/vdaas/vald/blob/master/CONTRIBUTING.md) document.
- [X] I have checked open [Pull Requests](https://github.com/vdaas/vald/pulls) for the similar feature or fixes?
- [ ] I have added tests and benchmarks to cover my changes.
- [ ] I have ensured all new and existing tests passed.
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: pkg/tools/cli/loadtest/config/config.go [hunk 1]
```diff
@@ -115,6 +115,7 @@ type Data struct {
 	Concurrency         int                `json:"concurrency" yaml:"concurrency"`
 	BatchSize           int                `json:"batch_size" yaml:"batch_size"`
 	ProgressDuration    string             `json:"progress_duration" yaml:"progress_duration"`
+	ResolveDNS          bool               `json:"resolve_dns" yaml:"resolve_dns"`
```
Comment (kpango) [inline pkg/tools/cli/loadtest/config/config.go:118]: We already have this option in `config.GRPCClient.ConnectionPool.ResolveDNS`

Comment (kmrmt) [reply pkg/tools/cli/loadtest/config/config.go]: Thank you for your comment.
I fixed it.

**Review by hlts2 — APPROVED** [review]
LGTM

**Review by kevindiu — APPROVED** [review]
LGTM

**Review by vankichi — DISMISSED** [review]
LGTM

**Review by kevindiu — APPROVED** [review]
LGTM

**Pull request merged**